### ISSUE TITLE
Phantom triplication of images in taxon media gallery

### ISSUE THREAD

**Issue by Archilegt** [NaturalHistoryMuseum/scratchpads2#6279]
Please, fix whatever is causing the phantom triplication of images in the taxon media gallery. For example, I added six images for one species and later edited them by adding additional information (taxonomic name field, description field). The six images exist only once inside the database, but they show triplicated (18 images) in the taxon page media gallery. See:
http://myriatrix.myspecies.info/taxonomy/term/38043/media

**Comment by Archilegt** [comment]
How wonderful. I opened one of each triplicated file again, this time from within the taxon's media gallery, clicked on the upper right corner gear, clicked on "Edit", did nothing but saving and voilá! Repeating that process, made all the phantoms disappear. But I hope that this won't be the way to solve this problem and that triplication is fixed from its source problem. Please!

**Comment by PaulKiddle** [comment]
Is this reproduceable? Does it happen every time you upload an image?

**Comment by alycejenni** [comment]
probably a duplicate of #6268
